.gitlab-ci.yml

```diff
@@ -212,6 +216,7 @@ build-linux-image:
       - ARCH: [amd64, arm64, ppc64le]
   retry: 2
   script:
+    - *docker-reader-role
```

Comment:
Is `DOCKER_PATH` env needed anywhere downstream? 

can it be just ?

```
- creds-helper init && eval $(creds-helper docker --eval "artifactory:v2/cloud/role/docker-reader-role")

Reply:
right, I simplified to use that instead.